opcodes.json

```diff
@@ -1437,24 +1437,42 @@
     "description": ""
   },
   "d0": {
-    "input": "",
-    "output": "",
-    "description": ""
+    "input": "offset",
+    "output": "data",
+    "description": "Load 32-byte word from data section of EOF container to the stack"
   },
   "d1": {
-    "input": "",
-    "output": "",
-    "description": ""
+    "input": "offset",
```

Comment:
not a stack input, but indeterminate argument

![image](https://github.com/user-attachments/assets/c67a3bc4-eea9-4f07-a5d0-c7ee313ba727)